include/sound/sof/info.h

```diff
@@ -31,6 +31,7 @@ enum sof_ipc_ext_data {
 	SOF_IPC_EXT_UNUSED		= 0,
 	SOF_IPC_EXT_WINDOW		= 1,
 	SOF_IPC_EXT_CC_INFO		= 2,
+	SOF_IPC_EXT_PROBE_INFO		= 3,
```

Comment:
Thanks @xiulipan , this is very informative -- short but has all the key pieces of info to understand what this is about.

One typo in commit message: "chagne" -> "change" -- recommend running codespell automatically on commits.

Reply:
thanks for the hint, let me recheck what goes wrong with my code spell.